annotation_suite_wrapper.sh

```diff
@@ -20,76 +20,90 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+source ../annotation-tools/annotation_suite_functions.sh
```

Comment:
Will this be an issue within the docker container when triggering this shell script via python code?

Reply:
Good point, not sure. Will have to try to test this in a docker container

Reply:
It doesn't appear to be an issue